src/components/QuickGrid/QuickGrid.Props.ts

```diff
@@ -49,7 +49,7 @@ export interface IQuickGridProps {
     customRowSorter?: (sortBy, sortDirection) => void;
     customCellRenderer?: (args: ICustomCellRendererArgs) => React.ReactNode;
     onGroupByChanged?: (groupBy: Array<IGroupBy>) => void;
-    groupRowFormat?: (rowData: any) => string;
+    groupRowFormat?: (rowData: any, columnName?: string) => string;
```

Comment:
Maybe change return type to React.Node so we have greater flexibility (it shouldn't break existing use cases)

Reply:
Fixed and tested for backward compatibility